src/ert/scheduler/job.py

```diff
@@ -111,6 +127,7 @@ async def _submit_and_run_once(self, sem: asyncio.BoundedSemaphore) -> None:
     async def __call__(
         self, start: asyncio.Event, sem: asyncio.BoundedSemaphore, max_submit: int = 2
     ) -> None:
+        self._max_submit = max_submit
```

Comment:
Why an attribute?

Reply:
It is only for the sake of the error message. The class `Job` does not hold the config, so it needs to keep track of `max_submit` from `__call__` in order to know how to make the error message. The attribute name can be anonymized further at least.